trainer/lib/trainer/test_parser.rb

```diff
@@ -212,9 +212,9 @@ def generate_cmd_parse_xcresult(path)
 
       # e.g. DEVELOPER_DIR=/Applications/Xcode_16_beta_3.app
       # xcresulttool version 23021, format version 3.53 (current)
-      match = `xcrun xcresulttool version`.match(/xcresulttool version (?<version>\d+),/)
-      version = match[:version]&.to_f
-      xcresulttool_cmd << '--legacy' if version >= 23_021.0
+      match = `xcrun xcresulttool version`.match(/xcresulttool version (?<version>[\d.]+)/)
```

Comment:
omitted `,` at the end because the code below this line would fail if the version includes any kind of alphabets(`[a-Z]`).

- with `,` at the end of regex

```
irb(main):005:0> match = 'xcresulttool version 22222.5.1-beta, Format Version 34'.match(/xcresulttool version (?<version>[\d.]+),/)
=> nil
```

- without `,` at the end of regex

```
irb(main):004:0> match = 'xcresulttool version 22222.5.1-beta, Format Version 34'.match(/xcresulttool version (?<version>[\d.]+)/)
=> #<MatchData "xcresulttool version 22222.5.1" version:"22222.5.1">
```